Diagnostics/HealthChecker/DataCollection/ServerInformation/Get-OperatingSystemInformation.ps1

```diff
@@ -68,7 +68,7 @@ Function Get-OperatingSystemInformation {
         -ScriptBlockDescription "Getting Current Time Zone" `
         -CatchActionFunction ${Function:Invoke-CatchActions}
     $osInformation.TLSSettings = Get-AllTlsSettingsFromRegistry -MachineName $Script:Server -CatchActionFunction ${Function:Invoke-CatchActions}
-    $osInformation.VcRedistributable = Get-VisualCRedistributableVersion
+    $osInformation.VcRedistributable = Get-VisualCRedistributableVersion -ComputerName $Script:Server
```

Comment:
Can we make this a parameter to Get-OperatingSystemInformation instead of assuming a script-scoped variable will be present somewhere?

Reply:
This problem is all over HealthChecker, so we will address it in a future PR focused on that.